packages/grafana-ui/src/components/DataLinks/DataLinkInput.tsx

```diff
@@ -132,6 +132,10 @@ export const DataLinkInput = memo(
 
       switch (event.key) {
         case 'Backspace':
+          if (stateRef.current.linkUrl.focusText.getText().length === 1) {
+            next();
+          }
+          // Fall through
```

Comment:
Do we need this comment?

Reply:
I thought it was a good practice to have a comment about intentional fall through. I will remove that comment and that space as well.